bin/do/setup

```diff
@@ -0,0 +1,263 @@
+#!/bin/sh
+
+# On Server
+# -------------------------------------------------
+GIT_URL="$1"
+HOST="$2"
+MONGO_PASS=$(date +%s | sha256sum | base64 | head -c 32 ; echo)
+
+sudo apt-get update
+sudo apt-get -y install language-pack-UTF-8
+sudo apt-get -y install libgmp3-dev
+sudo apt-get -y install imagemagick
+sudo apt-get -y install git-core
+sudo apt-get -y install nodejs
+
+## MongoDB 3
+# Add public key:
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+# Create a list file for MongoDB:
+echo "deb http://repo.mongodb.org/apt/ubuntu trusty/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
+# ​Reload local package database:
+sudo apt-get update
+echo "Installing Mongo DB"
+sudo apt-get install -y mongodb-org
+export LC_ALL=C
+# Create `rails` mongodb user in venice db:
+mongo --eval "conn = new Mongo(); db = conn.getDB('venice'); db.createUser({user:'rails', pwd:'$MONGO_PASS', roles:['readWrite']})"
+
+adduser rails --gecos "First Last,RoomNumber,WorkPhone,HomePhone" --disabled-password
```

Comment:
I think we should use --disable-login here, so rails user can't login to the system.